src/dotnet/APIView/APIViewWeb/Client/src/shared/signalr.ts

```diff
@@ -1,19 +1,25 @@
 import * as hp from "./helpers";
 
+let connection;
+// sender/server side of comment refresh 
+export function ReceiveComment(reviewId, elementId, partialViewResult) {
+  connection.invoke("ReceiveComment", reviewId, elementId, partialViewResult);
```

Comment:
Should we check to make sure the connection is active (IsConnected)?

Reply:
I couldn't find a function `connection.isConnected()` or similar - would `if (connection) { ... }` be enough?

Reply:
try `try `connection.state === signalR.HubConnectionState.Connected``